torchci/clickhouse_queries/commit_failed_jobs/params.json

```diff
@@ -1,3 +1,6 @@
 {
-  "shas": "String"
+  "params": {
+    "shas": "String"
+  },
+  "tests": []
```

Comment:
We will probably need a linter for this later I guess

Reply:
Added a simple linter for the structure but it doesn't check much more than that